src/data-root.ts

```diff
@@ -25,6 +29,27 @@ export async function lookup(
   }
 }
 
+/**
+ * Get the CID of a user's data root from the Fission server.
+ *
+ * @param username The username of the user that we want to get the data root of.
+ */
+export async function lookupOnFisson(
+  username: string
+): Promise<CID | null> {
+  try {
+    const resp = await fetch(`${setup.endpoints.api}/user/data/${username}`)
+    const cid = await resp.json()
```

Comment:
Does `resp.json()` work for plain-text responses? Or is `cid` an object?

Reply:
yup works for plain-text responses :+1: 

We could do `resp.text()` but that adds quotes on either side so it'd have to be something like
`(await resp.text()).replace('"', '')`

Reply:
I just figured this was cleaner

Reply:
Nice, didn't know that. Carry on 😄

Reply:
haha neither did I :smile: found out while coding this^